.circleci/config.yml

```diff
@@ -82,7 +90,6 @@ jobs:
               ccm start -v
               ccm status
               ccm checklogerror
-              ccm node1 nodetool status
               MAVEN_OPTS="-Xmx1g" mvn clean install
               # TODO: parallelise the following
               mvn surefire:test -Dtest=ReaperIT
```

Comment:
what do these changes have to do with this PR?

if it's general CircleCI stability improvements (eg i can see circleci is failing a bit recently) shouldn't it go in as a separate commit and PR?